android/gradle.properties

```diff
@@ -1,4 +1,4 @@
 Tor_kotlinVersion=1.3.50
-Tor_compileSdkVersion=28
-Tor_buildToolsVersion=28.0.3
-Tor_targetSdkVersion=28
+Tor_compileSdkVersion=30
```

Comment:
do projects that consume this lib need to up `minSdkVersion` to 30 as well..?

Reply:
no minSdkVersion remains 26. This is just the compile target which is a requirement (per google guidelines) to be at least 30 now